cookbooks/lib/features/basic_spec.rb

```diff
@@ -30,12 +30,22 @@
   end
 
   describe 'apt architecture' do
-    describe command('dpkg --print-architecture') do
-      its(:stdout) { should match(/amd64/) }
-    end
+    if os[:arch] =~ /ppc64/
+      describe command('dpkg --print-architecture') do
+        its(:stdout) { should match(/ppc/) }
+      end
+
+      describe command('dpkg --print-foreign-architectures') do
+        its(:stdout) { should match(//) }
```

Comment:
Is this supposed to be asserting the stdout is empty?

Reply:
Yes, on ppc64le there is output for this command.